src/lib.rs

```diff
@@ -2027,6 +2036,15 @@ pub trait Itertools: Iterator {
     /// assert_eq!(numbers.iter().find_or_first(|&&x| x > 5), Some(&1));
     /// assert_eq!(numbers.iter().find_or_first(|&&x| x > 2), Some(&3));
     /// assert_eq!(std::iter::empty::<i32>().find_or_first(|&x| x > 5), None);
+    ///
+    /// // An iterator of Results can return the first Ok or the last Err:
```

Comment:
This should be "or the first Err". Sorry for the back-and-forth, I did not realize it the first time.

Reply:
It's really something I overlooked. Thanks for reviewing!